### PR TITLE
updating readme

### PR DESCRIPTION
**Pull request by grantoesterling**
Updating readme with GitPOAP and changes from the past few months

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # ArtBot: The Art Blocks Discord Bot
 
-[![CircleCI](https://circleci.com/gh/ArtBlocks/artbot/tree/main.svg?style=svg)](https://circleci.com/gh/ArtBlocks/artbot/tree/main)
+[![GitPOAPs](https://public-api.gitpoap.io/v1/repo/ArtBlocks/artbot/badge)](https://www.gitpoap.io/gh/ArtBlocks/artbot)
```
Comment (jakerockland) [inline README.md:3]: Why don't we include both of these badges?

**Review by jakerockland — APPROVED** [review]
LGTM, with the one note that removing the Circle badge seems wrong – can't we just have both?

File: README.md (hunk 1 above)
Comment (grantoesterling) [reply README.md:3]: Because we don't use CircleCI for CI anymore, we use Github Actions now. Good point tho I'll add in a badge for Github Actions!!

Comment (jakerockland) [reply README.md:3]: Ah – roger that. Good call though ser yes pls.

**Pull request merged**